Library/Homebrew/test/os/mac/keg_spec.rb

```diff
@@ -5,28 +5,65 @@
 
   subject { described_class.new(keg_path) }
 
-  describe "#mach_o_files" do
-    let(:keg_path) { HOMEBREW_CELLAR/"a/1.0" }
+  let(:keg_path) { HOMEBREW_CELLAR/"a/1.0" }
+  let(:file) { keg_path/"lib/i386.dylib" }
 
-    before(:each) { (keg_path/"lib").mkpath }
-
-    after(:each) { subject.unlink }
+  before(:each) do
+    (keg_path/"lib").mkpath
+    cp dylib_path("i386"), file
+  end
 
-    it "skips hardlinks" do
-      cp dylib_path("i386"), keg_path/"lib/i386.dylib"
-      ln keg_path/"lib/i386.dylib", keg_path/"lib/i386_hardlink.dylib"
+  after(:each) { subject.unlink }
 
+  describe "#change_dylib_id" do
+    before(:each) do
```

Comment:
This can be moved into the toplevel `before`, can't it? Same with the one below.